src/box/lua/config/applier/console.lua

```diff
@@ -3,6 +3,9 @@ local console = require('console')
 local log = require('internal.config.utils.log')
 local instance_config = require('internal.config.instance_config')
 
+local old_socket_file = nil
+local old_socket = nil
```

Comment:
Isn't just a socket enough?

Reply:
I tried to do something like this (src/box/lua/config/applier/console.lua:apply)
```.lua
 local ok, res = pcall(console.listen, listen_uri)
 if old_socket ~= nil and old_socket ~= res then
    if type(old_socket.close) == "function" then
        old_socket:close()
    end
end
old_socket = res
```
But for some reason there are problems with calling config:reload() again. And also the test test/config-luatest/console_test.lua is falling apart.

Reply:
`socket.lua` specifies that socket has method `name`, which is implemented via `getsockname`. Try to use that